### ISSUE TITLE
[Feedback] Stats best winrate vs played move

### ISSUE THREAD

**Issue by ThomasCabaret** [leela-zero/leela-zero#1002]
Hi, I was wondering how often and how much leelaz in a setting comparable to its training settings can chose not to play moves with a highest winrate (according to its own playouts) than the actually played move.
I did the stats for 10 games (so 20 leela-player-instances). The leelaz settings are:
-network: a91721afa5cb92ec6c8efd77b8ac776b2d3a9acefedd324e58ca0e12376bc440
-playouts: 1600
-no randomization neither noise (of course).

You can find the game logs in attachments. with the interesting data grepped in a file, producing lines like that:

Diff: 5.52%, 
BasePlayouts: 535, BaseWin: 31.89%, BasePolicy: 73.74%, 
OpPlayouts: 182, OpWin: 37.41%, OpPolicy:  0.01%, 
MoveNum: 165

Should read: at move 165, a leela instance has chosen to play a 31.9% winrate move based on 535 playouts instead of another 37.4% winrate move based on 182 playouts.

Of course no clear conclusions can be directly deduced here, I am just sharing the raw data, some of the alternative options might just be moves too highly evaluated waiting to be downgraded with further playouts, but anyway here are some numbers:
-the average sum of winrate loss by leela-game-player is 20% (with probably a lot of false positive)
-discarding winrate diffs below 1% the average sum of winrate loss leela-game-player becomes 10%
-most of the loss comes from alternatives options with around 1.8% winrate diffs (for winrate diff below the frequency is higher but the loss is small, above the loss can be bigger, sometimes missing 5% on a move, but the frequency of this kind of "mistake" is really low)

My personal conclusion is that it's less a big deal than what I thought :) because in those kind of loss a lot happens when the game is already decided, and so, even a leela-game-player that would be able to get rid of them would not be significantly better. So I think we can be confident here than the UTC algo is using well (at least in a self consistent manner) the policy knowledge. But you can have a look at this raw data to make your own idea.

Regards

[OpGameLogs.zip](https://github.com/gcp/leela-zero/files/1796054/OpGameLogs.zip)
[grepOp.txt](https://github.com/gcp/leela-zero/files/1796055/grepOp.txt)

**Comment by sethtroisi** [comment]
closing, no active discussion
